grai-server/app/Dockerfile

```diff
@@ -35,6 +35,7 @@ RUN pip install "poetry==$POETRY_VERSION"
 RUN poetry config virtualenvs.create false  \
     && poetry lock --no-update \
     && poetry install --no-interaction --no-ansi --no-root --only main \
+    && poetry run pip install grai-source-snowflake --no-use-pep517 \
```

Comment:
Annoyingly I can't get this to work through poetry, without `--no-use-pep517` a dependent package tries to build from source and fails.